src/app/components/files/files.component.html

```diff
@@ -70,6 +70,7 @@
                   [format]="column.format"
                   [class]="column.class"
                   [sortable]="column.sortable"
+                  [sortingKey]="column.sortingKey"
```

Comment:
same as earlier, we can fallback and leave .json files as they are
```suggestion
                  [sortingKey]="column.sortingKey || column.key"
```